### ISSUE TITLE
Two strings are not translatable

### ISSUE THREAD

**Issue by bbb-user-de** [indico/indico#5467]
indico/modules/events/registration/models/registrations.py

```
88    __titles__ = [L_('Do not display to anyone'), L_('Display to other participants'), 
```

'Do not display to anyone' and  'Display to other participants' should be translatable.

**Comment by ThiefMaster** [comment]
Thanks, fixed in f857ffd23f - please ping me once you added the missing translations so I can pull them again prior to the 3.2 release.

**Comment by bbb-user-de** [comment]
@ThiefMaster translations added
